app/controllers/likes_controller.rb

```diff
@@ -0,0 +1,16 @@
+class LikesController < ApplicationController
+  def create
+    @post = Post.find(params[:post_id])
+    @like = @post.likes.build(user: current_user)
+    return unless @like.save
+
+    redirect_to user_post_path(@post.author, @post), notice: 'Liked!'
+  end
+
+  def destroy
+    @post = Post.find(params[:id])
+    @like = @post.likes.find_by(user: current_user)
+    @like&.destroy
+    redirect_to user_post_path(@post.author, @post), notice: 'Unliked!'
+  end
+end
```

Comment:
- [ ] Good Job for implementing a way user  able to add a like to posts as shown with the below screenshot.

![image](https://github.com/EvansSnave/Blog_app_rails/assets/93598189/1420cc72-9477-450a-ab4e-57dceb1e67f4)

![image](https://github.com/EvansSnave/Blog_app_rails/assets/93598189/dc29280e-1f69-419a-8be6-283709eeb7b8)